config/routes.rb

```diff
@@ -3,7 +3,7 @@
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :themes do
-    resources :bookings, only: [:new, :create]
```

Comment:
Adding CreateBookingTable on Theme show.thml.erb. Won't need the #new for bookings anymore.